2017/WG-09-12.md

```diff
@@ -0,0 +1,71 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the September 12th video call of WebAssembly's Working Group
+
+- **Host**: Google Hangouts
+- **Dates**: Tuesday September 12th, 2017
+- **Times**: 9:00am–10:00am Pacific Time
+- **Location**: *same Google Hangouts as previous CG video meetings*
```

Comment:
Seems odd to reuse the CG call ID if the intent is to have the first meeting with WG members only, no?

Reply:
I agree... would probably make more sense to have a different ID

Reply:
Ok, instead I'll invite WG members that have actually joined.